data/reference.json

```diff
@@ -225,6 +232,13 @@
         },
         "type": "boolean"
     },
+    "function_declaration.spacing_to_group": {
+        "description": "Whether to space a function name from following group.",
+        "example": {
+            "code": "if (true){}"
```

Comment:
Also here.

Reply:
Updated documentation to reflect the new functionality.